src/main/java/io/github/lottetreg/httpserver/BaseRoute.java

```diff
@@ -0,0 +1,27 @@
+package io.github.lottetreg.httpserver;
+
+public abstract class BaseRoute implements Routable {
+  protected String path;
+  protected String method;
+
+  BaseRoute(String path, String method) {
```

Comment:
Total nitpick, but I would personally find `new Route("GET", "/")` a little more intuitive, since it follows the format of an HTTP request (method first, then path).